etc/export_comCamSim.yaml

```diff
@@ -0,0 +1,19 @@
+datasets:
+- collections: refcats
+  datasetType: uw_stars_20240130
+- collections: skymaps
+  datasetType: skyMap
+# Workaround: use a matching expression rather than a specific
+# collection string for the collection argument, so to avoid
+# MissingCollectionError when the collection does not exist in
+# the target repo.
+# These are the non-raw collections in "u/jchiang/bfk_50240117_w_2024_04"
+- collections: "*u/jchiang/bfk_50240117_w_2024_04/20240131T061636Z"
+- collections: "*u/jchiang/ptc_50240117_w_2024_04/20240131T055006Z"
+- collections: "*u/jchiang/flat_50240117_w_2024_04/20240129T161508Z"
+- collections: "*u/jchiang/dark_50240117_w_2024_04/20240128T225538Z"
+- collections: "*u/jchiang/bias_50240117_w_2024_04/20240128T225209Z"
```

Comment:
As I understand it, this is just the first iteration of the sim. Is the plan to replace these later?

Reply:
Yes, that's my understanding too that this is only the first iteration and new set will be generated.  I assume only the final set will be copied into `/repo/embargo` where our prod deployment will use.  I don't think we really need to update the calibs in our dev central repo, because we aren't vetting the pipelines quality in dev but just ensure processing runs.   But, if we have doubts about prod for any reasons, we can update the dev repo too.